chromadb/segment/impl/vector/local_hnsw.py

```diff
@@ -287,9 +287,6 @@ def _apply_batch(self, batch: Batch) -> None:
             # If that succeeds, update the total count
             self._total_elements_added += batch.add_count
 
-            # If that succeeds, finally the seq ID
-            self._max_seq_id = batch.max_seq_id
```

Comment:
This is already updated here - https://github.com/chroma-core/chroma/pull/3064/files#diff-8a052898733bf8e0e1893bca365feb25d9abe61aa14197bb724f3583c181e461L304. So this really does nothing.